Update to include deployable module folder (#761)
terraform-ibm-modules/terraform-ibm-module-template: examples/advanced/README.md

# Advanced example

> <!-- There is a pre-commit hook that will take the title of each example add include it in the repos main README.md  -->

Reviewer: ```suggestion
<!-- For each example, a pre-commit hook adds a link with this title to the main README.md file  -->
```

> <!-- Add text below should describe exactly what resources are provisioned / configured by the example  -->

Reviewer: ```suggestion
<!-- Add text on the following lines to explain what the example demonstrates and what resources are provisioned or configured  -->
```
